config/database.yml

```diff
@@ -80,6 +80,6 @@ test:
 #
 production:
   <<: *default
-  database: mastodon_twitter_poster_production
-  username: mastodon_twitter_poster
-  password: <%= ENV['MASTODON_TWITTER_POSTER_DATABASE_PASSWORD'] %>
+  database: <%= ENV['DB_NAME'] || 'mastodon_twitter_poster_production' %>
+  username: <%= ENV['DB_USER'] || 'mastodon_twitter_poster' %>
+  password: <%= ENV['DB_PASS'] || ENV['MASTODON_TWITTER_POSTER_DATABASE_PASSWORD'] || '' %>
```

Comment:
Could you remove the `|| ''`? If the password is indeed empty, it can be exported as one of the empty variables. (even though I think it's kinda bad practice :)